### DISCUSSION TITLE
Need help on MapDriveInator Addon

### DISCUSSION THREAD

**Discussion by abc16361** [freeplane/freeplane#1815] [Help] [answered]
@EdoFro and the addon users,
After installing the MapDriveInator addon (MapDriveInator-v0.0.13.addon.mm), I created and saved a map with a folder link, as what the quick guide suggests. However, when I imported the MDI style using the command 'Import MDI Styles', it gave an error and couldn't continue the setup. I also tried it on FP portable version and the previous version of 1.11.12, all failed. The log shows that there is an error on class import, so is it a bug or are there some steps I missed on the setup? Thanks.
<img width="546" alt="importMDIStyle" src="https://github.com/freeplane/freeplane/assets/101362122/cedbaafd-a813-46ca-8999-eafcf167f9fa">
[log.zip](https://github.com/freeplane/freeplane/files/15206512/log.zip)

In order to continue the setup, I used  Menu Format -> Manage Styles -> Copy map styles from... to import the MDI styles, and it worked.

**Comment by Illegit** [comment]
In my case, this error happened when there were missing attributes or styles or link to a folder. It means the map parametrization wasn't complete... if I remember well.

**Reply by abc16361** [reply, marked as answer]
Thank you for the clue. After searching the discussion, I found out it needs to install [edofro.pseudofreeplaneapi.UserStyles library](https://github.com/freeplane/freeplane/discussions/922). Now it works perfect.
